lib/core/src/lib/styles/_mat-selectors.scss

```diff
@@ -125,6 +125,7 @@ $mat-button-touch-target: '.mat-mdc-button-touch-target';
 $mat-evolution-chip-text-label: '.mdc-evolution-chip__text-label';
 $cdk-overlay-pane: '.cdk-overlay-pane';
 $cdk-drag-preview: '.cdk-drag-preview';
+$cdk-drag-placeholder: '.cdk-drag-placeholder';
```

Comment:
I think the overall plan is to get rid of mat-selectors.scss file completely. So if you are depending on this selector somewhere in your design, it would be better to modify that code now to no longer use the selector, to avoid future effort

Reply:
and what would be the correct approach to use that selector? unfortunately, I can't replace it by ours, as it's internal cdk selector and it's impossible without it

Reply:
I replaced the occurrence of cdk-drag-placeholder with adf-datatable-row, and it seems to produce a similar layout to what is being produced right now. However, I would suggest to make sure that the styling does indeed match the required design

Reply:
Removed the selector